scripts/install_shiny_server.sh

```diff
@@ -2,6 +2,7 @@
 set -e
 
 SHINY_SERVER_VERSION=${1:-${SHINY_SERVER_VERSION:-latest}}
+UBUNTU_VERSION=${1:-${UBUNTU_VERSION:-18.04}}
```

Comment:
Could you remove this line like I commented?
https://github.com/rocker-org/rocker-versioned2/issues/550#issuecomment-1282621585

Reply:
Yes